docs/tutorial/add-subscription.md

```diff
@@ -0,0 +1,104 @@
+---
+id: add-subscription
+title: Add subscription
+sidebar_label: 9. Add subscription (advanced)
+---
+
+We discovered how to Query and Mutate datas, now, we are going to discover a most advanced part of GraphQL, the [event-based subscriptions](https://graphql.org/blog/subscriptions-in-graphql-and-relay/#event-based-subscriptions), well-known as `subscription` in the specification.
+
+In our tutorial, we will implement a subscription which will wait the end of the cooking process for a given recipe.
+
+During the cooking process, we will notify the user each second about the remaining time, and at the end, we will tell that the cooking time is over.
```

Comment:
we'll notify the user each second about the remaining time, and at the end, we'll notify that the cooking is over.